docs/developers/contributing.md

```diff
@@ -42,6 +42,26 @@ The following commands are now available from the repository root:
 
 More information can be found in [gulpfile.js](https://github.com/chartjs/Chart.js/blob/master/gulpfile.js).
 
+### Image-Based Tests
+
+Some display-related functionality is difficult to test via typical Jasmine unit test asserts. For this functionality, we have image-based tests that assert that the chart is drawn pixel-for-pixel matching an expected image.
```

Comment:
To avoid repeating `functionality`, what about:

> Some display-related functionality is difficult to test via typical Jasmine unit test, that's why we introduced image-based tests (#3988 and #5777) that assert that a chart is drawn pixel-for-pixel matching an expected image.

I would also link the two related PRs for more context.

*Edit:* I replaced my previous comment, seems I was translating `functionality` incorrectly.

Reply:
Updated. Good catch on having functionality twice